.env

```diff
@@ -0,0 +1 @@
+NEXT_PUBLIC_RPC_URL=https://polygon-mumbai.g.alchemy.com/v2/VB93DJmRg-zD-VUxpnChy0k87mr-P-EG
```

Comment:
@Adegbite1999 Let's add a `.env.example` file similar to this but without the actual keys.

Plus, we should add `.env` file to `.gitignore` so it's not tracked on version control